origin.spec

```diff
@@ -421,12 +415,7 @@ install -p -m 755 contrib/migration/* %{buildroot}%{_datadir}/%{name}/migration/
 %doc README.md
 %license LICENSE
 %{_bindir}/openshift
-%{_bindir}/kube-apiserver
```

Comment:
Do we need symlinks here for backwards compatibility?

Reply:
> Do we need symlinks here for backwards compatibility?

A symlink wouldn't work.  You could create a bash script if you really wanted to.  @smarterclayton didn't indicate it was a requirement.

Reply:
/cc @sdodson

Reply:
We don't have any dependencies on kube commands in the installer. This should be fine.